ui/lib/sync/addon/components/secrets/page/destinations/create-and-edit.ts

```diff
@@ -56,6 +56,20 @@ export default class DestinationsCreateForm extends Component<Args> {
         };
   }
 
+  groupSubtext(group: string, isNew: boolean) {
+    const dynamicText = isNew
+      ? 'used to authenticate with the destination'
+      : 'and the value cannot be read. Enable the input to update';
```

Comment:
```suggestion
  groupSubtext(group: string) {
    const dynamicText = this.args.destination.isNew
      ? 'used to authenticate with the destination'
      : 'and the value cannot be read. Enable the input to update';
```

Reply:
I went this exact same route, but it fails to read the args. It might be because I'm iterating over via a each and then each-in and then an each again, but passing in via the template solved the problem
![image](https://github.com/hashicorp/vault/assets/6618863/58b80ee9-9a6d-4850-86e8-634bf1c53472)

Reply:
strange! it must be how the template helper functions compile 🤔